config/locales/en.yml

```diff
@@ -1065,12 +1065,14 @@ Project attributes and sections are defined in the <a href=%{admin_settings_url}
               only_same_project_categories_allowed: "The category of a work package must be within the same project as the work package."
               does_not_exist: "The specified category does not exist."
             estimated_hours:
-              cant_be_inferior_to_remaining_work: "can't be inferior to 'Remaining work'."
-              must_be_set_when_remaining_work_is_set: "must be set when 'Remaining work' is set."
-              only_values_greater_or_equal_zeroes_allowed: "must be >= 0."
+              cant_be_inferior_to_remaining_work: "Cannot be lower than Remaining Work."
+              must_be_set_when_remaining_work_is_set: "Required when Remaining work is set."
+              only_values_greater_or_equal_zeroes_allowed: "Must be >= 0."
+              format: "%{message}"
             remaining_hours:
-              cant_exceed_work: "can't exceed 'Work'."
-              must_be_set_when_work_is_set: "must be set when 'Work' is set."
+              cant_exceed_work: "Cannot be higher than work."
+              must_be_set_when_work_is_set: "Required when Work is set."
+              format: "%{message}"
```

Comment:
What are these strings needed for?

Reply:
Hi @HDinger 
Sorry, I just copied the text from WP description, I will correct them.

According to [this blog post](https://www.bigbinary.com/blog/rails-6-allows-to-override-the-activemodel-errors-full_message-format-at-the-model-level-and-at-the-attribute-level?ref=writesoftwarewell.com), in rails the default format for showing validation error messages related to models' attributes is `%{attribute} %{message}`, but We can override the format of error messages by changing `errors.format`.